.github/workflows/tests_t8code_linkage_parallel_debug.yml

```diff
@@ -205,25 +205,25 @@ jobs:
     - name: check OpenCASCADE
       run: echo "Checking OpenCASCADE with MPI in debugging mode"
     - name: configure MPI OpenCASCADE debug
-      run: mkdir build_occ && cd build_occ && ../configure $CONFIG_DEBUG --with-occ CFLAGS="$CFLAGS_var -I/usr/include/opencascade" CXXFLAGS="$CXXFLAGS_var -I/usr/include/opencascade"
+      run: mkdir build_cad && cd build_cad && ../configure $CONFIG_DEBUG --with-cad CFLAGS="$CFLAGS_var -I/usr/include/opencascade" CXXFLAGS="$CXXFLAGS_var -I/usr/include/opencascade"
```

Comment:
How can this even work when the `configure` script does not accept `--with-cad`?

https://github.com/DLR-AMR/t8code/blob/44d96ab2dd2224c9a4cd935c47ef4eb108cbe0a9/config/t8_occ.m4#L15